bin/grouper-api

```diff
@@ -29,6 +30,34 @@ class Application(tornado.web.Application):
         super(Application, self).__init__(*args, **kwargs)
 
 
+class SendEmailThread(Thread):
+    def run(self):
+        while True:
+            logging.debug("Sending emails...")
+            try:
+                now = datetime.utcnow()
```

Comment:
For unit testing, I think it's preferable to split out the contents of this loop to a separate function that takes time as an argument.